test/controllers/DetermineTaxIdControllerSpec.scala

```diff
@@ -170,7 +170,43 @@ class DetermineTaxIdControllerSpec extends ItSpec {
       }
 
     }
+  }
+
+  "AuthenticatedActionRefiner bonus tests" - {
```

Comment:
bonus tests?

Reply:
Haha sorry, poorly named but I thought I'd sneak into this class as that's the controller being used to test - I didn't want to create a whole new spec just for a few tests to cover off AuthenticatedActionRefiner

Reply:
Maybe better to have an `AuthenticatedActionRefinerSpec` though (some time)

Reply:
Sure, I'll add now

Reply:
I'm not sure if this is a good example, but ... https://github.com/hmrc/vmv-frontend/blob/a53755bd9ec661597d615ed59bd587f2fb97f714/test/uk/gov/hmrc/vmv/actions/CcaAccountActionRefinerSpec.scala

Reply:
It does look like it, but Pawel is _very_ against mockito 😂

Reply:
is he against unit tests 🙂 ?